docroot/themes/humsci/humsci_basic/src/js/shared/navigation/secondary-toggler.js

```diff
@@ -9,8 +9,8 @@ import togglerHandler from './toggler-handler';
       if (togglers) {
         for (let i = 0; i < togglers.length; i += 1) {
           const toggler = togglers[i];
-          const togglerID = toggler.getAttribute('id');
-          const togglerContent = document.querySelector('[aria-labelledby="'.concat(togglerID, '"]'));
+          const togglerID = toggler.getAttribute('aria-controls');
+          const togglerContent = document.querySelector('[id="'.concat(togglerID, '"]'));
```

Comment:
Let's use this fix as an opportunity to modernize our code a bit:

```suggestion
          const togglerContent = document.querySelector(`[id="${togglerID}"]`));
```